### PR TITLE
Initial version of Java 11 extension.

### PR DESCRIPTION
**Pull request by mbooth101**
Signed-off-by: Mat Booth <mat.booth@redhat.com>

### CODE REVIEW

**Comment by mbooth101** [comment]
Looks like the build succeeded: https://flathub.org/builds/#/builders/2/builds/5189

**Comment by mbooth101** [comment]
What's next? Can it be merged?

**Comment by nedrichards** [comment]
I'd ideally like another reviewer to chime in and agree with me, but if they don't in the next day or so yes - this can be merged.

**Comment by mbooth101** [comment]
Cool, thanks :-)

File: org.freedesktop.Sdk.Extension.openjdk11.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Sdk.Extension.openjdk11</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>OpenJDK 11 SDK extension</name>
```
Comment (TingPing) [inline org.freedesktop.Sdk.Extension.openjdk11.appdata.xml]: `project_license`?

Comment (mbooth101) [reply org.freedesktop.Sdk.Extension.openjdk11.appdata.xml]: Okay I added licenses for everything in the source tree to be safe.

Comment (TingPing) [reply org.freedesktop.Sdk.Extension.openjdk11.appdata.xml]: I'd say those others should have a license too.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk11

**Comment by mbooth101** [comment]
Thanks!

**Comment by ctron** [comment]
I am still unable to make use of this:

~~~
➜  flathub git:(org.eclipse.iot.fourdiac.Ide) ✗ flatpak install --user flathub org.freedesktop.Sdk.Extension.openjdk11
error: Error searching remote flathub: Can't find ref org.freedesktop.Sdk.Extension.openjdk11
~~~

**Comment by mbooth101** [comment]
The build failed on arm, investigating.

**Pull request closed without merging**